docs/versions/netstandard2.0_ref.md

```diff
@@ -2585,6 +2794,9 @@
          public virtual Assembly EndInvoke(IAsyncResult result);
          public virtual Assembly Invoke(object sender, ResolveEventArgs args);
      }
+     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+     public struct RuntimeArgumentHandle
+     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
```

Comment:
Looks a bit weird that RuntimeArgumentHandle doesn't have `{ }`, probably a bug in the CCI writer that outputs just the skeleton but doesn't take into a count there might not be public members?